addons/website_forum/views/forum_forum_templates_post.xml

```diff
@@ -240,6 +247,13 @@
                         <t t-set="karma" t-value="question.karma_close if not question.can_close else 0"/>
                         <t t-set="classes" t-value="'btn-info'"/>
                     </t>
+                    <t t-if="user.karma > question.karma_unlink" t-call="website_forum.link_button">
+                        <t t-set="url" t-value="'/forum/' + slug(forum) + '/question/' + slug(question) + '/delete'"/>
+                        <t t-set="label">Delete</t>
+                        <t t-set="icon" t-value="'fa-trash'"/>
```

Comment:
```suggestion
                        <t t-set="icon">fa-trash</t>

Reply:
Won't this make it translatable?

Reply:
ahyes you are right, we can use t-valuef instead of t-value then :)